### PR TITLE
 Jedi Outcast files to ports/jedioutcast for Controls and Performance Optimisations

### PR DESCRIPTION
**Pull request by Habbening**
Jedi Outcast files to ports/jedioutcast for Controls and Performance Optimisations

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [ ] AmberELEC
- [ ] ArkOS
- [ ] ROCKNIX
- [X] muOS

### Resolution Tests
Test all major resolutions:
- [ ] 480x320
- [X] 640x480
- [ ] 720x720 (RGB30)
- [ ] Higher resolutions (e.g., 1280x720)

### CODE REVIEW

**Review by kloptops — APPROVED** [review]
👍

File: ports/jedioutcast/JediOutcast/openjo_sp.aarch64.gptk [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
-left_analog_down = s
-left_analog_left = a
-left_analog_right = d
-right_analog_up = mouse_movement_up
-right_analog_down = mouse_movement_down
-right_analog_left = mouse_movement_left
-right_analog_right = mouse_movement_right
+back =  n                                       # Quick Load
+start = m                                       # Quick Save
+guide = esc                                     # Pause / Menu
```
Comment (Cebion) [inline ports/jedioutcast/JediOutcast/openjo_sp.aarch64.gptk:3]: As we talked about most devices don't have a guide button so we need to add a alternative combo to get access to the esc button. 

Hotkey information can be found here

https://github.com/PortsMaster/gptokeyb#hotkey--button-for-additional-key-assignments

Hotkey by default is the back button + the hotkey you assigned.

**Review by Cebion — CHANGES_REQUESTED** [review]

File: ports/jedioutcast/JediOutcast/openjo_sp.aarch64.gptk (hunk 1 above)
Comment (JeodC) [reply ports/jedioutcast/JediOutcast/openjo_sp.aarch64.gptk:3]: `esc` should be assigned to `back`, and let the user reassign as needed. This is how it works in Harkinian ports.

Comment (Habbening) [reply ports/jedioutcast/JediOutcast/openjo_sp.aarch64.gptk:3]: yeah I just don't care about this anymore, removing the PR

**Pull request closed without merging**